addOns/zest/CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Changed
+- Clear Zest Results Panel when new script is opened.
```

Comment:
`opened` → `added` to be more clear. The commit/PR message/title could also be tweaked.

Reply:
Done